apps/vaccitron/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,5 @@
 export const environment = {
   production: true,
+  vaccinesDataUrl: 'fakeUrl',
+  reportsQueryInterval: 5 * 60 * 1000 // 5 minutes
```

Comment:
reportsQueryMinutesInterval

Reply:
It is in milliseconds actually. Would you like me to state it in the propname?

Reply:
Yes, applies to all the stack.